tests/foreman/cli/test_organization.py

```diff
@@ -444,60 +305,159 @@ def test_positive_add_user_by_id(self):
+        self.assertNotIn(user['login'], org['users'])
 
     @tier2
-    @stubbed()
     def test_positive_remove_user_by_name(self):
-        """@test: Create different types of users then add/remove user
-        by using the organization name
+        """Add an user to organization and remove it by its login and
+        organization name
 
         @feature: Organization
 
         @assert: The user is added then removed from the organization
+        """
+        org = make_org()
+        user = make_user()
+        Org.add_user({
+            'name': org['name'],
+            'user': user['login'],
+        })
+        Org.remove_user({
+            'name': org['name'],
+            'user': user['login'],
+        })
+        org = Org.info({'name': org['name']})
+        self.assertNotIn(user['login'], org['users'])
+
+    @tier2
+    def test_positive_add_admin_user_by_id(self):
+        """Create admin user and add it to organization by user and the
+        organization IDs
+
+        @feature: Organization
 
-        @status: manual
+        @assert: The user is added to the organization
         """
+        org = make_org()
+        user = make_user({'admin': '1'})
+        Org.add_user({
+            'id': org['id'],
+            'user-id': user['id'],
+        })
+        org = Org.info({'id': org['id']})
+        self.assertIn(user['login'], org['users'])
+        self.assertEqual(user['admin'], 'yes')
+
+    @tier2
+    def test_positive_add_admin_user_by_name(self):
+        """Create admin user and add it to organization by login and the
+        organization name
+
+        @feature: Organization
+
+        @assert: The user is added to the organization
+        """
+        org = make_org()
+        user = make_user({'admin': '1'})
+        Org.add_user({
+            'name': org['name'],
+            'user': user['login'],
+        })
+        org = Org.info({'name': org['name']})
+        self.assertIn(user['login'], org['users'])
+        self.assertEqual(user['admin'], 'yes')
+
+    @tier2
+    def test_positive_remove_admin_user_by_id(self):
+        """Create admin users then add user to organization and remove it by
+        using the organization name
+
+        @feature: Organization
+
+        @assert: The user is added then removed from the organization
+        """
+        org = make_org()
+        user = make_user({'admin': '1'})
+        Org.add_user({
+            'id': org['id'],
+            'user-id': user['id'],
+        })
+        Org.remove_user({
+            'id': org['id'],
+            'user-id': user['id'],
+        })
+        org = Org.info({'id': org['id']})
+        self.assertNotIn(user['login'], org['users'])
 
     @tier2
-    @stubbed()
     def test_positive_remove_admin_user_by_name(self):
-        """@test: Create admin users then add user and remove it
-        by using the organization name
+        """Create admin user then add user to organization and remove it by
```

Comment:
Since `org` is an approved acronym for `organization` as per our new naming conventions, similarly can you replace `organization` with `org` in all docstrings in this module.  This will shorten the docstring and make it easier :)